static/js/all.min.js.map

```diff
@@ -1 +1 @@
AAME,UAAUf,aAAa,aAAc,SAAU,CACrD0Q,kBAAmB,CACfpD,QAAS,yBACTH,QAAQ,EACRH,OAAQ,CACJ2D,cAAe,CACXrD,QAAS,cACTN,OAAQ,CAAC4D,4BAA6B,CAACtD,QAAS,YAAavD,MAAO,eAAgB8E,KAAMhO,MAAME,UAAUwP,aAC3GR,OAAQ,cAGnBlP,MAAME,UAAU8P,QAAUhQ,MAAME,UAAUf,aAAa,SAAU,MAAO,CACxE8Q,OAAQ,CACJxD,QAAS,4CACTJ,YAAY,EACZF,OAAQnM,MAAME,UAAUwP,WACxBxG,MAAO,sBACPoD,QAAQ,KAEZtM,MAAME,UAAUgQ,GAAKlQ,MAAME,UAAUwP,WACxC,SAAUzR,GACP,IAAIwK,EAAI,CACJ0H,SAAU,CAAC,CACP1D,QAAS,sBACTN,OAAQ,CACJgE,SAAU,CAAC,CAAC1D,QAAS,uBAAwBJ,YAAY,GAAK,WAC9DmD,OAAQ,0DACRC,SAAU,6FACVL,YAAa,oBAElB,CAAC3C,QAAS,sBAAuBH,QAAQ,EAAIH,OAAQ,CAACgE,SAAU,oBAAqB,gCAE5FlS,EAAEiC,UAAUkQ,KAAO,CACfC,QAAS,CAAC5D,QAAS,oCAAqCvD,MAAO,aAC/D+F,QAAS,CAACxC,QAAS,iBAAkBJ,YAAY,GACjD6C,OAAQ,CAAC,CACLzC,QAAS,+DACTJ,YAAY,EACZC,QAAQ,EACRH,OAAQ1D,GACT,CAACgE,QAAS,wDAAyDH,QAAQ,EAAIH,OAAQ1D,IAC1F0H,SAAU1H,EAAE0H,SACZZ,SAAY,CACR9C,QAAS,w7CACTJ,YAAY,GAEhBgD,QAAS,CACL5C,QAAS,wJACTJ,YAAY,GAEhBiD,QAAW,CAAC7C,QAAS,yCAA0CJ,YAAY,GAC3EoD,SAAU,uCACVL,YAAa,gCAEjB,IAAIxF,EAAInB,EAAE0H,SAAS,GAAGhE,OACtBvC,EAAEsF,OAASjR,EAAEiC,UAAUkQ,KAAKlB,OAAQtF,EAAY,SAAI3L,EAAEiC,UAAUkQ,KAAe,SAAGxG,EAAEyF,QAAUpR,EAAEiC,UAAUkQ,KAAKf,QAASzF,EAAE0F,QAAUrR,EAAEiC,UAAUkQ,KAAKd,QAAS1F,EAAE6F,SAAWxR,EAAEiC,UAAUkQ,KAAKX,SAAU7F,EAAEwF,YAAcnR,EAAEiC,UAAUkQ,KAAKhB,YAAanR,EAAEiC,UAAUoQ,MAAQrS,EAAEiC,UAAUkQ,KAnCvR,CAoCCpQ,OACFA,MAAME,UAAUqQ,OAAS,CACrBlB,QAAS,CACL5C,QAAS,8IACTJ,YAAY,GACb6C,OAAQ,gDAAiDD,QAAS,MAAOG,YAAa,6BAC1FpP,MAAME,UAAUsQ,WAAaxQ,MAAME,UAAUqQ,OAChDvQ,MAAME,UAAUuQ,GAAKzQ,MAAME,UAAU8J,OAAO,QAAS,CACjDqF,QAAS,sKACTqB,QAAS,2LACTpB,QAAW,gCACXG,SAAU,wFACVD,OAAQ,4DACRN,OAAQ,CAACzC,QAAS,mCAAoCH,QAAQ,YACvDtM,MAAME,UAAUuQ,GAAG,cAC9BzQ,MAAME,UAAUyQ,KAAO3Q,MAAME,UAAU8J,OAAO,QAAS,CACnDqF,QAAS,6VACTG,OAAQ,sFACRC,SAAU,CAAChD,QAAS,4FAA6FJ,YAAY,KAC7HrM,MAAME,UAAUf,aAAa,OAAQ,WAAY,CACjDyR,WAAY,CACR1H,MAAO,cACPuD,QAAS,eACTJ,YAAY,KAEhBrM,MAAME,UAAUf,aAAa,OAAQ,aAAc,CACnD0R,SAAU,CACNpE,QAAS,+CACTvD,MAAO,WACPiD,OAAQ,CAACkD,QAASrP,MAAME,UAAUyQ,KAAKtB,QAASD,YAAa,gBAGrEpP,MAAME,UAAU4Q,KAAO,CACnBC,SAAU,iCACV7B,OAAQ,CAACzC,QAAS,gCAAiCH,QAAQ,GAC3DkD,OAAQ,6DACRJ,YAAa,aACbK,SAAU,KACVH,QAAW,sBACX0B,KAAQ,aACThR,MAAME,UAAU+Q,MAAQjR,MAAME,UAAU4Q,KAC3C9Q,MAAME,UAAUgR,SAAWlR,MAAME,UAAU8J,OAAO,QAAS,CACvDqF,QAAS,sCACTqB,QAAS,8CACTS,UAAW,CAAC1E,QAAS,mHAAoHvD,MAAO,YAEpJlJ,MAAME,UAAUkR,OAAS,CACrBnC,QAAS,CAACxC,QAAS,eAAgBJ,YAAY,GAC/CgF,uBAAwB,CAAC5E,QAAS,sBAAuBH,QAAQ,EAAIpD,MAAO,UAC5EgG,OAAQ,CAACzC,QAAS,kCAAmCH,QAAQ,GAC7DiD,SAAY,CAAC9C,QAAS,4CAA6CJ,YAAY,GAC/E8C,aAAc,CAAC1C,QAAS,mBAAoBJ,YAAY,GACxDgD,QAAS,wLACTqB,QAAS,shBACTpB,QAAW,0BACXE,OAAQ,6FACRC,SAAU,uEACVL,YAAa,iBAEjBpP,MAAME,UAAUoR,KAAO,CACnBC,OAAQ,CACJ9E,QAAS,wFACTJ,YAAY,EACZnD,MAAO,UAEX+F,QAAS,MACTuC,IAAK,CAAC/E,QAAS,+EAAgFJ,YAAY,EAAInD,MAAO,UACtHuI,UAAW,CAAChF,QAAS,gBAAiBJ,YAAY,EAAInD,MAAO,aAC7DwI,SAAU,CACNjF,QAAS,2MACTJ,YAAY,EACZnD,MAAO,UAEXoG,QAAW,CAAC7C,QAAS,mEAAoEJ,YAAY,EAAInD,MAAO,aAChH8H,KAAQ,CAACvE,QAAS,+DAAgEJ,YAAY,EAAInD,MAAO,aACzGgG,OAAQ,CAACzC,QAAS,uFAAwFJ,YAAY,EAAIC,QAAQ,GAClIkD,OAAQ,CACJ/C,QAAS,gIACTJ,YAAY,GAEhB8B,IAAK,UACLwD,UAAW,YACXvC,YAAa"}
```

Comment:
This file doesn't belong in this PR, please remove.

Reply:
I am not sure, how it got added. Is there an easy way to removing it from the PR. One way I can think of is, git checkout the one from istio.io/master and re-commit/push. Any suggestions here ?

Reply:
Done and we shouldn't be seeing this file in the diff now.